pages/about.html

```diff
@@ -4,9 +4,20 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <!-- <link href="styles/style.css" rel="stylesheet"> -->
+  <link href="styles/about.css" rel="stylesheet">
```

Comment:
Your stylesheet wasn't linked to this page so it looks unstyled. The path in the href attribute needs to be updated to `/styles/about.css`